extensions/amp-story/ad-serving-integration.md

```diff
@@ -0,0 +1,129 @@
+# Ad serving integration for AMP Story
+
+## THIS IS A WORK-IN-PROGRESS
+
+This article is for parties who want to serve ads into AMP stories. If you're a
+publisher who wants to monetize your stories, go for [monetize with story](./monetize-with-story.md) instead,
```

Comment:
publisher who wants to monetize your stories, please refer to [monetize with story](./monetize-with-story.md) unless you want to serve ads by yourselves.